src/python/gudhi/cubical_complex.pyx

```diff
@@ -60,7 +60,7 @@ cdef class CubicalComplex:
 
         :param top_dimensional_cells: A multidimensional array of cells
             filtration values.
-        :type top_dimensional_cells: numpy ndarray
+        :type top_dimensional_cells: numpy ndarray or list of list of double
```

Comment:
Aren't they called `float` in python? I don't know what the standard python way is to document "or anything convertible to it". I would have kept just "numpy array", because list of double and list of list of list of double and list of array, etc will all work, but I guess having list of list of float as an example doesn't hurt.